pages/governance.md

```diff
@@ -0,0 +1,235 @@
+* [Benoît Legat (@blegat)](https://github.com/blegat)
+* [Joaquim Dias Garcia (@joaquimg)](https://github.com/joaquimg)
+* [Joey Huchette (@joehuchette)](https://github.com/joehuchette)
+* [Oscar Dowson (@odow)](https://github.com/odow)
+
+A new core contributor may be added by consensus of the current core
+contributors and notification to the Steering Council (see below).
+
+Before becoming a core contributor, it is expected that the community member is
+a repository maintainer (see below) of several repositories.
+
+### Emeritus Core Contributors
+
+Emeritus core contributors are community members who were core contributors, but
+are no longer active participants in the JuMP project.
+
+The emeritus core contributors are:
+
+* [Iain Dunning (@iainnz)](https://github.com/iainnz)
+
+### Repository Maintainers
+
+Repository maintainers are trusted members of the community who help the core
+contributors by managing a limited number of repositories in the
+[JuMP-dev Github organization].
+
+Any core contributor has individual authority to appoint a community member as a
+repository maintainer of a repository with notification to the other core
+contributors.
+
+Before becoming a repository maintainer, it is expected that the community
+member will have been an active participant in the development and maintenance
+of a repository for a sustained period of time. This includes triaging issues,
+proposing and reviewing pull requests, and updating any binary dependencies as
+needed.
+
+Community members may be maintainers of multiple repositories at the same time.
+
+Each repository should list the current maintainers in their corresponding
+`README.md` file.
+
+### NumFOCUS
+
+JuMP is a [fiscally sponsored project of NumFOCUS](https://numfocus.org/project/jump),
+a 501(c)(3) public charity in the United States. NumFOCUS provides JuMP with
+fiscal, legal, and administrative support to help ensure the health and
+sustainability of the project.
+
+### Steering Council
+
+The Steering Council supports the core contributors by representing JuMP in all
+interactions with NumFOCUS, e.g., by attending NumFOCUS's annual summit. In
+addition, the Steering Council:
+
+* Approves expenditures related to JuMP and paid through the NumFOCUS account.
+* Negotiates and approves contracts between NumFOCUS and external contractors
+  who provide paid work to JuMP.
+
+The current members of the Steering Council are:
+
+* [Miles Lubin (Google)](https://github.com/mlubin)
+* [Juan Pablo Vielma (Google)](https://github.com/juan-pablo-vielma)
+* [Joey Huchette (Rice University)](https://github.com/joehuchette)
+* [Oscar Dowson](https://github.com/odow)
+* [Changhyun Kwon (U. South Florida)](https://github.com/chkwon)
+
+If a member of the Steering Council leaves the Steering Council, the remaining
+members, in consultation with the core contributors, will invite a member of the
+community to join in order to maintain a quorum of five members.
+
+#### Conflict of Interest
```

Comment:
I would move "Conflict of interest" under the "decision making process" section and extend it to all decisions made by the steering council or core contributors. Core contributors should also disclose conflicts of interest with other core contributors and step back from decisions when conflicts of interests are in play.